tests/admin_docs/tests.py

```diff
@@ -463,3 +454,17 @@ def test_initial_header_level(self):
         header = 'should be h3...\n\nHeader\n------\n'
         output = utils.parse_rst(header, 'header')
         self.assertIn('<h3>Header</h3>', output)
+
+
+@unittest.skipUnless(utils.docutils_is_available, "no docutils installed.")
+class AdminDocViewFunctionsTests(TestDataMixin, AdminDocsTestCase):
+
+    def test_simplify_regex(self):
+        tests = (
+            ('^a', '/a'),
+            ('^(?P<a>\w+)/b/(?P<c>\w+)/$', '/<a>/b/<c>/'),
+            ('^(?P<a>\w+)/b/(?P<c>\w+)$', '/<a>/b/<c>'),
+            ('^(?P<a>(x|y))/b/(?P<c>\w+)$', '/<a>/b/<c>')
```

Comment:
Include trailing comma so if more lines are added later, we don't have to modify this line again.
Another failing case: `('^(?P<a>(x|y))/b/(?P<c>a\w+)a', '/<a>/b/<c>a')`